.github/actions/e2e-setup-common/action.yaml

```diff
@@ -26,6 +26,10 @@ runs:
         go-version-file: "tests/e2e/go.mod"
         cache: true
 
+    - name: Install Ginkgo CLI
+      shell: bash
+      run: go install github.com/onsi/ginkgo/v2/ginkgo@latest
```

Comment:
Might be better to pin the version to avoid any risk of incompatibility in the future (unless using `v2` guarantees a stable API, but I'm not sure)

Reply:
As we are using v2(major version), I am going by SEMVER rule, any minor patch will be backward compatible.
They have been introducing a lot of fixes so I like to keep the latest. 

So going with SEMVER standard rules, it should guarantee a stable API, you might correct me or they might prove me wrong at some point. But till then I would like to get the latest fixes.